test/testdata/resolver/fuzz_type_member_forget.rb

```diff
@@ -9,4 +9,4 @@ class DifferentArityChild < Parent # error: Type `TParent` declared by parent `P
   TChild = type_member(fixed: String)
 end
 
-T.cast(1, DifferentArityChild[Integer, String, Symbol]) # error-with-dupes: Wrong number of type parameters for `DifferentArityChild`. Expected: `0`, got: `3`
+T.cast(1, DifferentArityChild[Integer, String, Symbol]) # error-with-dupes: Wrong number of type parameters for `DifferentArityChild`. Expected: `1`, got: `3`
```

Comment:
The type was previously fixed to `T.untyped` because it was missing. Since we don't fix it anymore, the arity is now `1`.